src/main/java/software/amazon/awssdk/crt/http/HttpLibrary.java

```diff
@@ -0,0 +1,25 @@
+package software.amazon.awssdk.crt.http;
+
+import software.amazon.awssdk.crt.CrtResource;
+
+import java.io.Closeable;
+
+/**
+ * This initializ
+ */
+public class HttpLibrary extends CrtResource implements Closeable {
```

Comment:
This class isn't necessary, just bake the http library init/clean up into the existing crtInit() and shutdown functions

Reply:
Fixed